telegram/_chatadministratorrights.py

```diff
@@ -163,9 +185,9 @@ def __init__(
         can_edit_messages: Optional[bool] = None,
         can_pin_messages: Optional[bool] = None,
         can_manage_topics: Optional[bool] = None,
-        can_post_stories: Optional[bool] = None,
-        can_edit_stories: Optional[bool] = None,
-        can_delete_stories: Optional[bool] = None,
+        can_post_stories: bool = None,  # type: ignore # noqa: RUF013
+        can_edit_stories: bool = None,  # type: ignore # noqa: RUF013
+        can_delete_stories: bool = None,  # type: ignore # noqa: RUF013
```

Comment:
why not keep it as `Optional[bool]`?

Reply:
I guess this makes it a tad more obvious that these should in fact be passed now to type checkers. And probably works beter with the attribute assignments below. I'm fine either way

Reply:
Yeah exactly my thinking

Reply:
but we already raise a `TypeError` when they are `None`. This just confuses the type checker and `test_official`. IDE's will still show these to be optional anyway.

Reply:
test_official failing due to this would indeed be a reason to change IMO. the fewer special cases in ther, the better

Reply:
agreed with test_official argument